Unhandled exception in thread started by 
sys.excepthook is missing
lost sys.stderr
